client/components/pie-chart/README.md

```diff
@@ -60,6 +61,7 @@ export default class Example extends Component {
 				data={ data }
 				title={ translate( '%(total)s Total Searches', { args: { total } } ) }
 			/>
+			<PieChartLegend data={ data } />
```

Comment:
I think you need to wrap those 2 component in a `<div></div>` otherwise React would complain about this.